extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -88,6 +88,9 @@ tags: {  # <amp-date-countdown>
     value_casei: "continue"
     value_casei: "stop"
   }
+  attrs: {
```

Comment:
Please alphabetize by name of the attribute. This should go above `end-date`.

Please also put `name` and `value` on separate lines.

Reply:
Thanks @honeybadgerdontcare !  I have update the ordering of this new attribute and moved `name` and `value` to separate lines.  I have also updated the validator test and output files.  Thanks!